pallets/basic-sto/src/lib.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2020 Polymath
+
+//! # STO Module
```

Comment:
Elaborate bit here?

Reply:
This is intentionally blank since this is a POC module with no real-world use case. I guess I'll add this in the description itself.